src/arithmetic/int_set.cc

```diff
@@ -529,6 +554,22 @@ class IntervalSetEvaluator :
     return Combine<T>(analyzer_, a, b);
   }
 
+  template<typename T>
+  inline IntervalSet VisitDivExpr_(const T* op) {
+    IntervalSet a = this->Eval(op->a);
+    IntervalSet b = this->Eval(op->b);
+    if ((MatchPoint(a, op->a) && (MatchPoint(b, op->b) || SelfBoundedVar(b, op->b)))
+          || (SelfBoundedVar(a, op->a) && SelfBoundedVar(b, op->b))) {
+      // e.g.,
```

Comment:
This part is a bit confusing. If the divisor is already a set(which means it is relaxed), we should not return the original var.

Reply:
but in `Everything<Div>` such case evaluates to `IntervalSet::Everything()`. 
e.g., `4 / tvm.var() => 4 / tvm.var()`, while `4 / tvm.shape_var() => (-inf, +inf)`

Reply:
i think this can be fixed by having shape_var return single_pt

Reply:
yeah... you're right.